internal/pkg/kwatchman/kwatchman.go

```diff
@@ -1,11 +1,25 @@
 package kwatchman
 
 import (
-	"github.com/snebel29/kwatchman/internal/pkg/cli"
+	log "github.com/sirupsen/logrus"
 	"github.com/snebel29/kwatchman/internal/pkg/watcher"
+	"os"
+	"os/signal"
+	"syscall"
 )
 
-func Run(c *cli.CLIArgs) error {
-	w := watcher.NewWatcher(c)
+var shutdown chan os.Signal
+
+func Start(w watcher.Watcher) error {
```

Comment:
My original thinking was to have a `main` as slim as possible, and full coverage beyond that, I think I might refactor this part and push everything here back to main to avoid this unnecessary abstraction.

Thanks!